docs/zkapps/o1js/on-chain-values.mdx

```diff
@@ -23,7 +23,7 @@ Please note that zkApp programmability is not yet available on Mina Mainnet, but
 
 In a zkApp, you can access the current [on-chain state](/zkapps/how-to-write-a-zkapp#on-chain-state) and many other on-chain values of the account.
 
-The zkApp account's on-chain state can be updated by account updates included in a transaction (see [Tutorial 10: Account Updates]((/zkapps/tutorials/account-updates)). On the Mina blockchain, each zkApp account provides eight fields of ~32 bytes each of arbitrary storage for the on-chain state.
+The zkApp account's on-chain state can be updated by account updates included in a transaction (see [Tutorial 10: Account Updates](/zkapps/tutorials/account-updates)). On the Mina blockchain, each zkApp account provides eight fields of ~32 bytes each of arbitrary storage for the on-chain state.
```

Comment:
```suggestion
The zkApp account's on-chain state can be updated by account updates included in a transaction (see [Tutorial 10: Account Updates](/zkapps/tutorials/account-updates). On the Mina blockchain, each zkApp account provides eight fields of ~32 bytes each of arbitrary storage for the on-chain state.
```

Reply:
Hi @barriebyron ,

I think, we both removed one one parenthesis, and now it's missing one ')' 😅:
<img width="890" alt="Screenshot 2024-02-18 at 10 18 41 PM" src="https://github.com/o1-labs/docs2/assets/55076843/64d24946-dbf4-48d6-841d-8a619507c6cd">

I will add fix for it while working on this issue #760 .


Thank You 😃